modules/lvms-upgrading-lvms-on-sno.adoc

```diff
@@ -6,7 +6,7 @@
 [id="lvms-upgrading-lvms-on-sno_{context}"]
 = Upgrading {lvms} on {sno} clusters
 
-Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.11 to {lvms} 4.12 on {sno} clusters.
+Currently, it is not possible to upgrade from {rh-storage} Logical Volume Manager Operator 4.12 to {lvms} 4.13 on {sno} clusters.
```

Comment:
@suleymanakbas91 Is this correct?

Reply:
No, it is not correct. Upgrade from 4.11 (tech preview) to 4.12 was not possible due to breaking changes. But, it is possible to upgrade from 4.12 to 4.13.